_posts/2023-05-15-odh-release-1.6.0-blog.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: blog
+author: lavlas
+title:  Open Data Hub 1.6.0 Release Guide
+preview: Open Data Hub 1.6.0 was just released
+date: 2023-05-15
+categories: features, road map, release
+---
+
+What is new?
+------
+Changes included in the Open Data Hub v1.6.0 release:
+* ODH Dashboard [v2.9](https://github.com/opendatahub-io/odh-dashboard/releases/tag/v2.9.0) 
+* Jupyter Notebooks [v1.4.0](https://github.com/opendatahub-io/notebooks/releases/tag/v1.4.0)
+* Data Science Pipelines Operator [v0.2.0](https://github.com/opendatahub-io/data-science-pipelines-operator/releases/tag/v0.2.0)
+* [Model Mesh](https://github.com/opendatahub-io/modelmesh-serving)
+* [TrustyAI Explainability](https://github.com/trustyai-explainability/trustyai-explainability)
+* Incubation of [Distributed Workloads](https://github.com/opendatahub-io/distributed-workloads) stack 
+  * `CodeFlare SDK` to define and control remote distributed compute jobs and infrastructure with any Python based environment
+  * `Multi-Cluster Application Dispatcher (MCAD)` for management of batch jobs
+  * `Instascale` for on-demand scaling of a Kubernetes cluster
+  * `KubeRay` for management of remote Ray clusters on Kubernetes for running distributed compute workloads
```

Comment:
```suggestion
* Incubation of [Distributed Workloads](https://github.com/opendatahub-io/distributed-workloads) stack 
  * Ease of use with `Codeflare SDK`: The Codefare SDK is integrated into the out-of-the-box ODH notebook images and provides an interactive client for data scientists to define resource requirements (GPU, CPU, and memory) and to submit and manage training jobs.
  * Batch Management with `Multi-Cluster Application Dispatcher (MCAD)`: MCAD manages a queue of training jobs that data scientists submit. MCAD ensures that jobs are not started until all required compute resources are available on the cluster. MCAD ensures that a given team has not requested more aggregate resources than their quota allows, and ensures that highest priority jobs are executed first. Finally, MCAD ensures that all processes necessary to execute a distributed run are scheduled concurrently, meaning that compute cycles aren’t wasted waiting for processes to be scheduled.
  * Dynamic scaling with `Instascale`: Instascale works alongside MCAD to ensure that the OpenShift cluster contains sufficient compute resources to execute a job. Instascale optimizes compute costs by launching the right-sizes compute instance for a given job, and releasing these instances when they are no longer needed.
  * `KubeRay` for management of remote Ray clusters on Kubernetes for running distributed compute workloads
```

Reply:
Sounds good. I opened https://github.com/opendatahub-io/opendatahub.io/pull/121 for a blog post.